ts/state/ducks/theme.tsx

```diff
@@ -9,7 +9,7 @@ export const applyTheme = (theme: ThemeStateType) => {
   };
 };
 
-export const initialThemeState: ThemeStateType = 'light';
+export const initialThemeState: ThemeStateType = 'classic-dark';
```

Comment:
just want to make sure we double check that we want this to be the default

Reply:
Answered above 👍